modules/oe_translation_cdt/modules/oe_translation_cdt_mock/oe_translation_cdt_mock.module

```diff
@@ -0,0 +1,100 @@
+<?php
+
+/**
+ * @file
+ * Primary module hooks for OpenEuropa Translation CDT mock module.
+ */
+
+declare(strict_types=1);
+
+use Drupal\Core\Entity\Display\EntityViewDisplayInterface;
+use Drupal\Core\Entity\EntityInterface;
+use Drupal\Core\Url;
+use Drupal\oe_translation_cdt\Api\CdtApiWrapperInterface;
+use Drupal\oe_translation_cdt\TranslationRequestCdt;
+
+/**
+ * Implements hook_entity_ENTITY_TYPE_view_alter().
+ */
+function oe_translation_cdt_mock_oe_translation_request_view_alter(array &$build, EntityInterface $entity, EntityViewDisplayInterface $display) {
+  if ($entity->bundle() !== 'cdt') {
+    return;
+  }
+
+  /** @var \Drupal\oe_translation_cdt\TranslationRequestCdtInterface $entity */
+  if (!isset($build['meta'])) {
+    return;
+  }
+  $row = &$build['meta']['#rows'][0];
+
+  $operations = [];
+  $all_statuses = [
+    CdtApiWrapperInterface::STATUS_REQUEST_COMPLETED => 'Completed',
+    CdtApiWrapperInterface::STATUS_REQUEST_IN_PROGRESS => 'In progress',
+    CdtApiWrapperInterface::STATUS_REQUEST_CANCELLED => 'Cancelled',
+    CdtApiWrapperInterface::STATUS_REQUEST_PENDING_APPROVAL => 'Pending approval',
+    CdtApiWrapperInterface::STATUS_REQUEST_UNDER_QUOTATION => 'Under quotation',
+  ];
+  foreach ($all_statuses as $status => $label) {
+    $operations[$status] = [
+      'title' => sprintf('Update status to %s (mock)', $label),
+      'url' => Url::fromRoute('oe_translation_cdt_mock.request_status', [
```

Comment:
Not sure why but when using the "In progress" update option, there is no information logged in the request. Afaik we do not have an ongoing status for our requests but we should still log the update in the request logs. Same thing happens for the job updates

Reply:
We've got only these states of request in the CDT:

- COMP - The request has been completed and delivered.
- INPR - The request is in progress or request for quotation approved.
- CANC - The request has been cancelled
- PEND - The request quotation is ready to be approved by the client (Requests for Quotation only).
- UNDE - The request quotation is under quotation.

So I have no way to distinguish "requested" from "in progress".

The log is only written when there is an actual change. When you start the translation, it is already in INPR state.